chia/plotting/manager.py

```diff
@@ -162,7 +162,8 @@ def process_file(file_path: Path) -> Dict:
             if file_path.exists():
                 if (
                     file_path in self.failed_to_open_filenames
-                    and (time.time() - self.failed_to_open_filenames[file_path]) > 1200
+                    and (time.time() - self.failed_to_open_filenames[file_path])
+                    < self.refresh_parameter.retry_invalid_seconds
                 ):
                     # Try once every 20 minutes to open the file
```

Comment:
this comment is outdated with this patch

Reply:
Right, changed!